mautic/mautic
# Fix entity lookup type

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
We noticed issue with lookup type If use two different lookup type from same model. This cause issue in segment email translations




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create segment email (must be segment)
3. Create second segment email as translation and set Is a translation of  email create in step 
4. After save see **Is a translation** of  is empty
![image](https://user-images.githubusercontent.com/462477/107194593-889d2680-69f0-11eb-8f79-60e6a406a5a1.png)
5. With this PR works properly
